go.mod

```diff
@@ -8,7 +8,7 @@ require (
 	github.com/Masterminds/sprig v2.22.0+incompatible
 	github.com/fatih/color v1.9.0 // indirect
 	github.com/ghodss/yaml v1.0.0
-	github.com/giantswarm/apiextensions/v3 v3.14.1
+	github.com/giantswarm/apiextensions/v3 v3.16.2-0.20210202133116-302f4a6aacb5
```

Comment:
Is there a saner release of `apiextensions` that can be used?

Reply:
Not yet. My other PR to apiextensions is hanging.

Reply:
Excellent